reconcile/openshift_base.py

```diff
@@ -185,8 +194,12 @@ def apply(dry_run, oc_map, cluster, namespace, resource_type, resource):
         # do not skip if this is a cluster scoped integration
         if namespace != 'cluster' and not oc.project_exists(namespace):
             msg = f"[{cluster}/{namespace}] namespace does not exist (yet)"
-            logging.warning(msg)
-            return
+            if wait_for_namespace:
+                logging.info(msg)
```

Comment:
we will be up 3,5 minutes waiting here. can we add a msg about that? somthing like `msg += ' - waiting a bit'`

Reply:
added!